R/app.R

```diff
@@ -120,7 +120,7 @@
 #' to \code{true}, or a timeout happens. It returns \code{TRUE} is the
 #' expression evaluated to \code{true}, possible after some waiting.
 #'
-#' \code{find_widget()} finds the corresponding HTML element of a Shiny
+#' \code{app$find_widget()} finds the corresponding HTML element of a Shiny
 #' widget. It returns a \code{\link{widget}} object.
 #'
 #' \code{expect_update()} is one of the main functions to test Shiny apps.
```

Comment:
I agree with you, the user should know that `expect_update()` is a package-level function, and that's the function you should use in the readme/vignette. But since this right here is the documentation for shinyapp, in order to be consistent, I think it's better to show `app$expect_update()`, and mention that a more convenient notation for the same function is `expect_update()`. But both should be documented in my opinion.